lighthouse-core/report/v2/report-styles.css

```diff
@@ -602,6 +602,38 @@ summary.lh-passed-audits-summary::-webkit-details-marker {
 
 .lh-table {
   --image-preview-size: 24px;
+  border: 1px solid var(--report-secondary-border-color);
+  border-collapse: collapse;
+  table-layout: fixed;
+  width: 100%;
+}
+
+.lh-table thead {
+  background: #ccc;
```

Comment:
`--report-secondary-border-color` has less weight. If you don't want to overload `--report-secondary-border-color`, maybe define a new var.